### DISCUSSION TITLE
Agregar botones extra de Z y X para la versión móvil

### DISCUSSION THREAD

**Discussion by heikomaru** [ppy/osu#14458] [Ideas] [answered]
Quisiera que agregen una especie de botones extra para los de versión móvil,no pooseemos teclado así que se nos dificulta el querer realizar los mapas arriba de 3 estrellas

**Comment by bdach** [comment]
Use English, please.

**Reply by heikomaru** [reply]
I would like you to add a kind of extra buttons for the mobile version, we do not have a keyboard so it is difficult for us to want to make the maps above 3 stars

**Comment by Game4all** [comment, marked as answer]
This is tracked at  #4201
